torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -641,7 +641,11 @@ py::object toPyObject(IValue ivalue) {
     for (const auto i : c10::irange(list.size())) {
       t[i] = toPyObject(IValue{list.get(i)});
     }
+#if __cplusplus >= 202002L
+    return t
```

Comment:
missing semicolon; is this being tested?

Reply:
Sorry, the quick fixes have now been corrected and verified.